platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/event/validation/DefaultInternalEventValidator.java

```diff
@@ -174,8 +175,9 @@ private boolean areRequiredFieldsNonNull(@NonNull final GossipEvent event) {
      */
     private boolean isTransactionByteCountValid(@NonNull final GossipEvent event) {
         int totalTransactionBytes = 0;
-        for (final ConsensusTransaction transaction : event.getHashedData().getTransactions()) {
-            totalTransactionBytes += transaction.getSerializedLength();
+        final Iterator<Transaction> iterator = event.transactionIterator();
```

Comment:
```
event.transactionStream()
    .mapToInt(t -> t.getSerializedLength())
    .sum();
```
That would look way better from my point of view.

Reply:
see comment https://github.com/hashgraph/hedera-services/pull/13761#discussion_r1636234187